enterprise/fulltext-addon/src/test/java/org/neo4j/kernel/api/impl/fulltext/integrations/bloom/BloomFulltextIndexBenchmarks.java

```diff
@@ -35,17 +35,13 @@
 import org.neo4j.kernel.internal.GraphDatabaseAPI;
 import org.neo4j.test.rule.TestDirectory;
 
-import static org.neo4j.kernel.api.impl.fulltext.integrations.bloom.BloomFulltextConfig.bloom_enabled;
-
-@Ignore( "These are rudimentary benchmarks, but implemented via the jUnit framework to make them easy to run " +
-         "from an IDE." )
+@Ignore( "These are rudimentary benchmarks, but implemented via the jUnit framework to make them easy to run " + "from an IDE." )
```

Comment:
i'm sure i already mentioned this before: BloomFulltextIndexBenchmarks should not be Ignored and live in the product. It should be moved into benchmarks repo and run there.

Reply:
Oh sorry, this is an old class that has just been moved around. These tests were used as an tool when developing the old fulltext index, will clean up.